recipes/cc65/all/conanfile.py

```diff
@@ -0,0 +1,188 @@
+from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
+from conans.client.tools.win import msvs_toolset
```

Comment:
This import... this is not public API. If it is needed, maybe we can add it as a `tool` (is this the best name so far?)

Reply:
This recipe also needs the toolset in its `package_info`: https://github.com/conan-io/conan-center-index/pull/1191

Reply:
We are introducing this as a tool, but we can improve the recipe in the future: https://github.com/conan-io/conan/issues/7061

Reply:
For future reference https://github.com/conan-io/conan/pull/7134